src/components/schedule/index.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./style.sass";
+import moment from "moment";
+
+function Schedule({ name, schedules }) {
+  return (
+    <article className="schedule-component-container">
+      <section className="cinema-name">{name}</section>
+      <section className="schedule-bound">
+        {schedules.length !== 0 &&
```

Comment:
```suggestion
        {schedules.length &&
```

Reply:
Nếu không có so sánh trực tiếp thì khi schedules.length = 0 nó sẽ in ra màn hình số  0 anh ạ